### ISSUE TITLE
Noob question

### ISSUE THREAD

**Issue by jgsimard** [TobiaMarcucci/gcspy#1]
Hi when I run the notebooks, I get the error
SolverError: Either candidate conic solvers (['GLPK_MI', 'SCIPY']) do not support the cones output by the problem (SOC, NonNeg, Zero), or there are not enough constraints in the problem.
What should I do ?

**Comment by TobiaMarcucci** [comment]
Hello! First of all sorry if the code is still very experimental and not well documented/tested.

It seems that you are trying to solve a mixed integer second order cone program and you don't have a solver installed for that. If you don't have them, I suggest you download Gurobi or Mosek (they both support misocp). They are free for academic users. Let me know if this helps!

**Comment by jgsimard** [comment]
Hi, thanks for the input, it workd now! I just started learning learning the whole convex optimization / gcs field!

**Comment by yslusa** [comment]
Hello @jgsimard ,

I also have a noob question, I have mosek solver installed in my User/Home folder (I am using macOS) as instructed on Mosek website. I cloned this repo to my User/Home folder and I started to work on this project in a virtual environment, I installed all required packages (numpy, cvxpy, matplotlib, and etc.), then what should I do with the solver? I tried to "pip install mosek" but it still has the same SolverError.

I wonder what did you do to make it work?
